app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/ConfirmPledgeDetailsScreen.kt

```diff
@@ -565,22 +605,40 @@ fun BonusSupportContainer(
                 Spacer(modifier = Modifier.width(dimensions.paddingMediumSmall))
             }
 
-            Text(
-                modifier = Modifier
-                    .background(
+            Row(
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier.background(
                         color = colors.kds_white,
                         shape = shapes.small
-                    )
-                    .padding(
-                        start = dimensions.paddingMediumSmall,
-                        top = dimensions.paddingMediumSmall,
-                        bottom = dimensions.paddingMediumSmall,
-                        end = dimensions.paddingMediumSmall
-                    ),
-                text = totalBonusAmount,
-                style = typography.headline,
-                color = colors.textAccentGreen
-            )
+                ).padding(
+                    start = dimensions.paddingXSmall,
+                    top = dimensions.paddingXSmall,
+                    bottom = dimensions.paddingXSmall,
+                    end = dimensions.paddingXSmall
+                ),
+
+            ) {
+                Text(
+                    text = currencySymbolAtStart ?: "",
+                    color = colors.kds_create_700
+                )
+                BasicTextField(
+                    modifier = Modifier.width(IntrinsicSize.Min),
+                    value = if (totalBonusSupport % 1.0 == 0.0) totalBonusSupport.toInt().toString() else totalBonusSupport.toString(),
+                    onValueChange = {
+                        if (it.length <= bonusAmountMaxDigits) onBonusSupportInputted(it.parseToDouble())
+                    },
+                    textStyle = typography.title1.copy(color = colors.kds_create_700),
+                    keyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
+                    cursorBrush = SolidColor(Color.LightGray)
```

Comment:
@mtgriego If I needed this to change for Light/Dark theme what would I change?

Reply:
Pretty much anything you want to change with the system theme needs to use the theme colors we created.
In this case I see you are using out theme colors for the text (but use `textAccentGreen` here as well), but the cursor brush is using a defined color.  Instead of `Color.LightGray` you should use one of our colors (in this case perhaps` iconSubtle`. 
 Not all of our color names match their use cases yet).